pkg/virt-controller/watch/snapshot/restore.go

```diff
@@ -817,6 +826,14 @@ func setLastRestoreAnnotation(restore *snapshotv1.VirtualMachineRestore, obj met
 	obj.GetAnnotations()[lastRestoreAnnotation] = getRestoreAnnotationValue(restore)
 }
 
+func CreateRestorePVCDefFromVMRestore(vmRestoreName, restorePVCName string, volumeSnapshot *vsv1.VolumeSnapshot, volumeBackup *snapshotv1.VolumeBackup, sourceVmName, sourceVmNamespace string) *corev1.PersistentVolumeClaim {
+	pvc := CreateRestorePVCDef(restorePVCName, volumeSnapshot, volumeBackup)
+	pvc.Labels[restoreSourceNameLabel] = sourceVmName
+	pvc.Labels[restoreSourceNamespaceLabel] = sourceVmNamespace
```

Comment:
the labels/annotations maps are created in `CreateRestorePVCDef` if nil.  That should be done in this function since no labels/annotations are added there